packages/server/src/api/controllers/plugin/index.ts

```diff
@@ -75,14 +84,9 @@ export async function create(ctx: any) {
     const doc = await pro.plugins.storePlugin(metadata, directory, source)
 
     clientAppSocket?.emit("plugins-update", { name, hash: doc.hash })
-    ctx.body = {
```

Comment:
Nice find!

Reply:
It fell out after I typed the response object.